pvlib/pvsystem.py

```diff
@@ -831,7 +834,7 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
 
```

Comment:
Thanks for the detailed explanation! So does this refer to reference 2 in the Docstring (I don't have access so cannot check)?